.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,6 +34,9 @@ If you have a stack trace to include, we recommend putting inside a `<details>`
 </details>
 
 **Information about your Sapper Installation:**
+
+<!-- please include the output of npx envinfo --system --npmPackages svelte* --binaries --browsers -->
```

Comment:
This should include sapper. Also I would make it more consistent with the other bullets and remove the duplicate items. I.e.

- The output of: npx envinfo --system --npmPackages svelte,sapper --binaries --browsers

- Your browser and the version: (e.x. Chrome 52.1, Firefox 48.0, IE 10)

- Your hosting environment: (i.e. Local, GCP, AWS, Now, etc...)

- If it is an exported (npm run export) or dynamic application.

- Whether your application uses Webpack or Rollup